### DISCUSSION TITLE
Allow larger profile pics

### DISCUSSION THREAD

**Discussion by mperham** [community/community#127147] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Profile images are currently limited to 1MB in size, which seems incredibly skimpy in today’s world of 24MP phone cameras. Bump this up to 5 or 10MB so we can upload our phone pics without a lot of pre processing on our part?

![IMG_0014](https://github.com/community/community/assets/2911/2c9de723-d446-4f89-a162-a6cec5cf86e0)

**Comment by ebndev** [comment]
@mperham thanks for bringing this up!
